test/gtest/ucp/test_ucp_dt.cc

```diff
@@ -181,6 +181,44 @@ class test_ucp_dt_iter : public ucs::test_with_param<ucp_datatype_t> {
 
 ucp_dt_generic_t* test_ucp_dt_iter::dt_gen = 0;
 
+/* The following tests reuse the generic datatype and its packed size will be
+ * set after the first packing. */
```

Comment:
is it still relevant after adding buffer,count?

Reply:
No, it is not. I removed the comment.